README.md

```diff
@@ -525,5 +525,7 @@ gitstatus are released. This may change in the future but not soon.
 
 ## License
 
-GNU General Public License v3.0. See [LICENSE](LICENSE). Contributions are covered by the same
+Most of the repository is covered by GNU General Public License v3.0. See [LICENSE](LICENSE). Contributions are covered by the same
 license.
+
+`gitstatus.prompt.sh` and `gitstatus.prompt.zsh` are part of the public domain.
```

Comment:
This doesn't work for code. From https://en.wikipedia.org/wiki/Public-domain-equivalent_license:

> Public domain equivalent licenses exist because some legal jurisdictions do not provide for authors to voluntarily place their work in the public domain [...]

Reply:
What about this?
```
`gitstatus.prompt.sh` and `gitstatus.prompt.zsh` are covered by [Unlicense](https://unlicense.org).
```

Reply:
"Covered" is unclear. I also believe the copy of the license file must be put in the repo.

It doesn't seem like you have experience with these things. For the sake of saving time, please research this properly and get the PR in a state that will look obviously correct. Supply all the context that you deem necessary to convince me that the change is correct.

It might be more practical for you to relicense your own code.

Reply:
The current license requires any use of the code to use the same license.

"Permissions of this strong copyleft license are conditioned on making available complete source code of licensed works and modifications, which include larger works using a licensed work, **under the same license**. Copyright and license notices must be preserved. Contributors provide an express grant of patent rights."

Reply:
> The current license requires any use of the code to use the same license.

I know. Hence if you release your code under GPLv3, you'll be compliant. I'm saying it might be easier than to attempt to relicense *my* code.

Reply:
I'd rather keep my zsh theme to myself than use GPLv3 and have to deal with stuff like this:
```
If the program does terminal interaction, make it output a short
notice like this when it starts in an interactive mode:

    <program>  Copyright (C) <year>  <name of author>
    This program comes with ABSOLUTELY NO WARRANTY; for details type `show w'.
    This is free software, and you are welcome to redistribute it
    under certain conditions; type `show c' for details.

The hypothetical commands `show w' and `show c' should show the appropriate
parts of the General Public License.  Of course, your program's commands
might be different; for a GUI interface, you would use an "about box".
```

Shouldn't `gitstatus.prompt.(z)sh` also do this?

Reply:
That’s not a requirement.